src/static/helpers/slasClient.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import type { OperationOptions } from "retry";
+import { Response, ClientConfig } from "@commerce-apps/core";
+import type { RequestInit } from "node-fetch";
+
+type LoginRequest = {
+  client_id?: string;
+  response_type?: string;
+  redirect_uri: string;
+  state?: string;
+  scope?: string;
+  usid?: string;
+  channel_id: string;
+  code_challenge: string;
+} & { [key: string]: any };
+
+export type TokenRequest = {
+  refresh_token?: string;
+  code?: string;
+  usid?: string;
+  grant_type: string;
+  redirect_uri?: string;
+  code_verifier?: string;
+  client_id?: string;
+} & { [key: string]: any };
+
+export type TokenResponse = {
+  access_token: string;
+  id_token: string;
+  refresh_token: string;
+  expires_in: number;
+  token_type: string;
+  usid: string;
+  customer_id: string;
+  enc_user_id: string;
+} & { [key: string]: any };
```

Comment:
Aren't these types defined in the generated `shopperLogin.ts`? You could import them, but you'd also have to be careful about circular dependencies.

Reply:
`If linting is reporting an error, then you should fix the error, rather than disabling linting. In this case, the comment blocks are not the expected format, so they should be changed. (They are jsdoc, not tsdoc. There are minor but annoying differences.)`

Responding to your linting here cause I couldn't find an easy way to reply that's my b:

So usually I try to fix linting errors as they arise rather than disabling linting. Here's an example of the way it's currently documented:

```javascript
/**
 * A single function to execute the ShopperLogin Public Client Guest Login with proof key for code exchange flow as described in the [API documentation](https://developer.salesforce.com/docs/commerce/commerce-api/guide/slas-public-client.html).
 * @param slasClient a configured instance of the ShopperLogin SDK client.
 * @param parameters - parameters to pass in the API calls.
 * @param parameters.redirectURI - Per OAuth standard, a valid app route. Must be listed in your SLAS configuration. On server, this will not be actually called
 * @param parameters.usid? - Unique Shopper Identifier to enable personalization.
 * @returns TokenResponse
 */
export async function loginGuestUser(
  slasClient: ISlasClient,
  parameters: {
    redirectURI: string;
    usid?: string;
  }
):
```

The linter is complaining about the dot notation, which I believe is the most intuitive way to write and understand the documentation. Also from what I've seen/researched so far (including JSDoc), this style of is pretty standard on documenting parameters with properties. I'd also like to keep the documentation changes as close as to the `isomorphic sdk` as well just for consistency. I'm totally cool with changing the documentation if you have a convincing argument for the other side though 👍

Reply:
Do what you want, but if there's just one rule in one file that's causing trouble and you don't want to fix it, then just disable the one rule in the one file, not every rule in every file. 😉

Reply:
Totally fair point. I figured since `renderedTemplates` was in `.eslintignore` I could toss `dist` as well since it's also dynamically generated.